src/context/TodoContext.tsx

```diff
@@ -0,0 +1,77 @@
+/* eslint-disable @typescript-eslint/no-unused-vars */
+import React, { ReactNode, useState, useMemo } from 'react';
+
+import { useLocalStorage } from '../hooks/useLocalStorage';
+import { Status } from '../types/Status';
+import { Todo } from '../types/Todo';
+
+type Props = {
+  children: ReactNode;
+};
+
+export const TodoContext = React.createContext({
+  todos: [] as Todo[],
+  status: Status.All,
+});
+
+export const TodoUpdateContext = React.createContext({
+  addTodo: (_todo: Todo) => { },
+  deleteTodo: (_todoId: string) => { },
+  deleteCompletedTodos: () => { },
+  updateTodo: (_todo: Todo) => { },
+  changeTodosStatus: () => { },
+  setStatus: (_status: Status) => { },
+  isTodosCompleted: false,
+});
+
+export const TodoProvider: React.FC<Props> = ({ children }) => {
```

Comment:
check `PropsWithChildren` property in the `'react'`
you can use it like

```
import { FC, PropsWithChildren } from 'react'

.....
export const TodoProvider: FC<PropsWithChildren<{}>> = ({ children }) => {
```

Reply:
Really thanks for the review, it was very helpful for me!